### ISSUE TITLE
[Race Driver: GRID] In the Replay Music, two songs are playing at the same time.

### ISSUE THREAD

**Issue by Lettendo** [NC22/EEnot-Audio-extractor#1]
Files: "_replay1.wav" and "_replay2.wav"

**Comment by antoniu200** [comment]
This project seems dead.
You can import the songs in Audacity and the issue should be solved by changing the number of input channels. Be careful, the songs are "U-Law" (in Audacity) at 32,000 Hz.

**Comment by Lettendo** [comment]
Yeah, I think I had already figured that out back then but thanks for the reply!

**Comment by GeraltV47** [comment]
so what have i do to listen to music from race driver grid? all 8 songs have sound layers played at the same time i mean the harder last lap bass part is being heard all the time and i cannot separate these layers that change their volume depending what happens in the game

**Comment by Lettendo** [comment]
Use Audacity to import the game's music files using the settings that antoniu200 stated.
You can then rearrange the different tracks as you like and export them as AAC or example (very efficient codec)
I personally didn't mix the different tracks up, but just left them as they were.
